addons/mail/models/mail_activity_plan.py

```diff
@@ -22,7 +22,8 @@ def _get_model_selection(self):
     company_id = fields.Many2one(
         'res.company', default=lambda self: self.env.company)
     template_ids = fields.One2many(
-        'mail.activity.plan.template', 'plan_id', string='Activities')
+        'mail.activity.plan.template', 'plan_id', string='Activities',
+        copy=True)
```

Comment:
Could be great to add an unit test for that so that it is covered as a feature for any future refactoring